### DISCUSSION TITLE
New Contributor

### DISCUSSION THREAD

**Discussion by peculiaruc** [angolaosc/.github#2] [General]
Hi 
I am Peculiar, am new here. I will like to contribute to this community, learn how things works here, also help and guide others.

**Comment by fgonga** [comment]
2

**Reply by euotiniel** [reply]
Yes, it will be interesting to work in the open sourse world with more Angolans

**Reply by peculiaruc** [reply]
Does this mean non-Angolan can not join this community?

**Reply by antonio-pedro99** [reply]
Hi guys, I am Antonio Pedro, core leader of this community. Everyone is welcome to be part of this community :100:

**Comment by antonio-pedro99** [comment]
Hi @peculiaruc, please keep up to date with our organization on GitHub, soon we are going to release our social media handlers with more information about this community.

**Comment by peculiaruc** [comment]
Great.

@Antonio, I am willing to work with you in this community, thanks for your
feedback
